lib/handlers/healthmonitor.js

```diff
@@ -5,6 +5,14 @@ var MongoClient = require('mongodb').MongoClient;
 var request = require('request');
 var ditchHelper = require('../util/ditchhelper.js');
 var util = require('util');
+var dns = require('dns');
+var dfc;
+
+if (!fhconfig.value('openshift3')) {
+  var fhdfc = require('fh-dfc');
+  dfc = fhdfc(fhconfig.value('fhdfc'));
+  dfc.apps
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ASemicolon)